scripts/js/020-components/08-print.js

```diff
@@ -0,0 +1,23 @@
+// track print events in Google Analytics
+
+var printedCount = 0;
+
+function logPrintEvent() {
+	printedCount++;
+	if (printedCount < 2) {
```

Comment:
Why do you send a print event when less than 2 have been fired? Shouldn't it just always send print events?

Reply:
This is weird thing with Chrome that respects both the window.onbeforeprint as well as the Safari only option.